bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/handler/It600Handler.java

```diff
@@ -0,0 +1,265 @@
+            }
+            return;
+        }
+
+        logger.debug("Does not know how to handle command `{}` ({}) on channel `{}`!", command,
+                command.getClass().getSimpleName(), channelUID);
+    }
+
+    private void handleCommandForWorkType(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            findLongProperty("ep_9:sIT600TH:HoldType", "HoldType").map(DeviceProperty.LongDeviceProperty::getValue)
+                    .map(value -> switch (value.intValue()) {
+                        case AUTO -> "AUTO";
+                        case MANUAL -> "MANUAL";
+                        case TEMPORARY_MANUAL -> "TEMPORARY_MANUAL";
+                        case OFF -> "OFF";
+                        default -> {
+                            logger.warn("Unknown value {} for property HoldType!", value);
+                            yield "AUTO";
+                        }
+                    }).map(StringType::new).ifPresent(state -> updateState(channelUID, state));
+            return;
+        }
+
+        if (command instanceof StringType typedCommand) {
+            long value;
+            if ("AUTO".equals(typedCommand.toString())) {
+                value = AUTO;
+            } else if ("MANUAL".equals(typedCommand.toString())) {
+                value = MANUAL;
+            } else if ("TEMPORARY_MANUAL".equals(typedCommand.toString())) {
+                value = TEMPORARY_MANUAL;
+            } else if ("OFF".equals(typedCommand.toString())) {
+                value = OFF;
+            } else {
+                logger.warn("Unknown value `{}` for property HoldType!", typedCommand);
+                return;
+            }
+            var property = findLongProperty("ep_9:sIT600TH:SetHoldType", "SetHoldType");
+            if (property.isEmpty()) {
+                return;
+            }
+            cloudApi.setValueForProperty(dsn, property.get().getName(), value);
+            return;
+        }
+
+        logger.debug("Does not know how to handle command `{}` ({}) on channel `{}`!", command,
+                command.getClass().getSimpleName(), channelUID);
+    }
+
+    private Optional<DeviceProperty.LongDeviceProperty> findLongProperty(String name, String shortName) {
+        var deviceProperties = findDeviceProperties();
+        var property = deviceProperties.stream().filter(p -> p.getName().equals(name))
+                .filter(DeviceProperty.LongDeviceProperty.class::isInstance)
+                .map(DeviceProperty.LongDeviceProperty.class::cast).findAny();
+        if (property.isEmpty()) {
+            property = deviceProperties.stream().filter(p -> p.getName().contains(shortName))
+                    .filter(DeviceProperty.LongDeviceProperty.class::isInstance)
+                    .map(DeviceProperty.LongDeviceProperty.class::cast).findAny();
+        }
+        if (property.isEmpty()) {
+            logger.warn("{}/{} property not found!", name, shortName);
```

Comment:
```suggestion
            logger.debug("{}/{} property not found!", name, shortName);
```

You only need to warn the user if there is something they should really know or fix.